camera_cards

```diff
@@ -213,21 +216,24 @@ if [[ "${AIP}" == "YES" ]] ; then
     elif [[ "${CAMERA_CARD_TYPE}" == "GENERAL" ]] ; then
         CONCATENATED_VIDEO_FILE="${AIPDIR}/objects/${MEDIAID}_concatenated.${FIRST_FILE#*.}"
     fi
+    STREAM_MD5="${METADATA_OUTPUT_DIR}/$(basename ${CONCATENATED_VIDEO_FILE})_bitstream_md5.txt"
     # create list of files to concatenate, formatted for ffmpeg
     while read FILE ; do
         echo "file '${FILE}'" >> "${TEMP_CONCATLIST}"
     done <"${TEMP_VIDEOLIST}"
     # concatenate video files in the order they are printed in $TEMP_CONCATLIST; map metadata from the first video file (in sequence) onto the concatenated file
-    ffmpeg -f concat -safe 0 -i "${TEMP_CONCATLIST}" -i "${FIRST_FILE}" -map 0 -map_metadata 1 -c copy "${CONCATENATED_VIDEO_FILE}"
-    _writelog "CONCATENATED VIDEO FILE" "${CONCATENATED_VIDEO_FILE}"
+    ffmpeg -f concat -safe 0 -i "${TEMP_CONCATLIST}" -i "${FIRST_FILE}" -map 0 -map_metadata 1 -c copy "${CONCATENATED_VIDEO_FILE}" -f md5 -map 0:a? -map 0:v -c copy "${STREAM_MD5}"
```

Comment:
streamhash?

Reply:
Changed in commit: bcc5315f2a962870bb7d226c54eb611fddf56c45